appengine/standard-storage/Gemfile

```diff
@@ -20,7 +20,7 @@ gem "sinatra", "~>2.1"
 # [END gae_standard_storage_dependencies]
 
 group :test do
-  gem "capybara"
+  gem "capybara", "~>3.38.0"
```

Comment:
Do you mean to pin to 3.38.x or is 3.39+ also okay?

Reply:
`3.38.0` has the changes we need. I meant anything above or equal to `3.38.0`

Reply:
So maybe better `~> 3.38` so that it will also allow 3.39.x, 3.40.x, etc.